### PR TITLE
Create OWNERS

### PR DESCRIPTION
**Pull request by npm1**
Manual addition of this file upstream seems to be required.

### CODE REVIEW

**Comment by npm1** [comment]
@spanicker

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-24 19:36:35
Finished: 2017-08-24 19:40:56





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17743)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/268107975)

**Comment by Hexcles** [comment]
Seems like GitHub has a different syntax for OWNERS files: each line is a GitHub handle instead of an email address (e.g. https://github.com/w3c/web-platform-tests/blob/master/dom/OWNERS)

So it should probably look like:
```
@npm1
```

File: longtask-timing/OWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+panicker@chromium.org
```
Comment (spanicker) [inline longtask-timing/OWNERS]: add yourself or Tim as well?

**Comment by npm1** [comment]
Oh maybe that's why we don't pull the OWNERS files, the formatting difference? I added myself, ptal

**Comment by Hexcles** [comment]
@npm1 that's correct. The syntax and content of OWNERS are different on GitHub and Chromium. We do not pull nor push OWNERS changes.

**Review by spanicker — APPROVED** [review]

**Review by foolip — APPROVED** [review]

**Pull request merged**